QKSMS/build.gradle

```diff
@@ -33,7 +24,7 @@ android {
         release {
             minifyEnabled true
             shrinkResources true
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'qksms-proguard.pro'
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'qksms-proguard.pro'
```

Comment:
`proguard-android` is the default config. It makes proguard shrink your apk in a conservative way. This means removing unused classes, methods, etc.

`proguard-android-optimize` adds optimizations and obfuscation on top. Optimizations are kinda nasty, because they sometimes introduce bugs. And they really don't improve things much. And obfuscation is only ever needed if you want to make your apk harder to decompile. Which doesn't make sense in FOSS.